synapse/server_notices/server_notices_manager.py

```diff
@@ -64,14 +65,22 @@ async def send_notice(
             is_state_event: Is the event a state event
             txn_id: The transaction ID.
         """
-        room_id = await self.get_or_create_notice_room_for_user(user_id)
+        (room_id, room_created) = await self.get_or_create_notice_room_for_user(user_id)
```

Comment:
Instead of passing the `room_created` flag back, would it make sense to call `_update_notice_user_profile_if_changed` inside of `get_or_create_notice_room_for_user`?

We seem to set it for creating a room, so seems like it would make sense.

Reply:
done.